fenix/app/src/test/java/org/mozilla/fenix/home/privatebrowsing/DefaultPrivateBrowsingControllerTest.kt

```diff
@@ -60,12 +60,14 @@ class DefaultPrivateBrowsingControllerTest {
 
     @Test
     fun `WHEN private browsing learn more link is clicked THEN open support page in browser`() {
+        val learnMoreURL = SupportUtils.getGenericSumoURLForTopic(SupportUtils.SumoTopic.PRIVATE_BROWSING_MYTHS) +
```

Comment:
Let's hardcode the expected canonical URL here. That way if we change the implementation of `getGenericSumoURLForTopic` this test will fail and it can signal to us that this also needs to get fixed.